firstvoices/backend/models/immersion_labels.py

```diff
@@ -22,10 +22,10 @@ class Meta:
             )
         ]
         rules_permissions = {
-            "view": predicates.is_visible_object,
-            "add": predicates.is_language_admin_or_super,
-            "change": predicates.is_language_admin_or_super,
-            "delete": predicates.is_language_admin_or_super,
+            "view": predicates.has_visible_site,
+            "add": predicates.can_add_core_uncontrolled_data,
+            "change": predicates.can_edit_core_uncontrolled_data,
+            "delete": predicates.can_delete_core_uncontrolled_data,
```

Comment:
This is a big permission change that I don't think we want to make. Immersion Labels are not "core" uncontrolled data-- that was meant to cover media.

Our roles and permissions doc lists Immersion Labels as only editable by Language Admins and Superadmins: https://firstvoices.atlassian.net/wiki/spaces/FIR/pages/220790795/User+Roles+in+fv-be

Reply:
View permission looks good though